src/main/resources/application.yml

```
@@ -1,3 +1,31 @@
+      hibernate:
+        format_sql: true
+        highlight_sql: true
+        use_sql_comments: true
+springdoc:
+  default-produces-media-type: application/json;charset=UTF-8
+logging:
+  level:
+    org.hibernate.orm.jdbc.bind: TRACE
+    org.hibernate.orm.jdbc.extract: TRACE
+    org.springframework.transaction.interceptor: TRACE
+    com.back: DEBUG
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)